src/FormCollection/Resources/views/form_theme_table.html.twig

```diff
@@ -0,0 +1,52 @@
+{%- block button_add -%}
+    {%- set attrDataAction = (attr['data-action']|default('') ~ ' ' ~ controllerName ~ '#add')|trim -%}
+    <tr>
+        <td colspan="2">
+            <button data-action="{{ attrDataAction }}"
+                    class="{{ button_add.class|default('') }}" type="button">{{ button_add.text|default('Add')|trans }}</button>
```

Comment:
Same comments as previous form theme

Reply:
OK with the changes of the default options.